docs/Data/Json/JTable.md

```diff
@@ -3,19 +3,19 @@
 #### `noStyle`
 
 ``` purescript
-noStyle :: TableStyle
+noStyle :: forall p. TableStyle p
```

Comment:
Nah, it's unrelated :) I just needed this to be open-ended within the module it was defined.